test/Core.Test/Models/Business/Tokenables/HCaptchaTokenableTests.cs

```diff
@@ -3,12 +3,44 @@
 using Bit.Core.Entities;
 using Bit.Core.Models.Business.Tokenables;
 using Bit.Core.Tokens;
+using Bit.Test.Common.AutoFixture.Attributes;
 using Xunit;
 
 namespace Bit.Core.Test.Models.Business.Tokenables
 {
     public class HCaptchaTokenableTests
     {
+        [Fact]
+        public void CanHandleNullUser()
+        {
+            var token = new HCaptchaTokenable(null);
+
+            Assert.Equal(default, token.Id);
+            Assert.Equal(default, token.Email);
+        }
+
+        [Fact]
+        public void TokenWithNullUserIsInvalid()
+        {
+            var token = new HCaptchaTokenable(null)
+            {
+                ExpirationDate = DateTime.UtcNow + TimeSpan.FromDays(1)
+            };
+
+            Assert.False(token.Valid);
+        }
+
+        [Theory, BitAutoData]
+        public void TokenValidityCheckNullUserIsFalse(User user)
```

Comment:
Is this testing a null UserId?
```suggestion
        public void TokenValidityCheckNullUserIdIsInvalid(User user)
```

Reply:
There are two `TokenIsValid` methods on the class. One is an override method from the `ExpiringTokenable` class and the other is a method intended to be used on 2fa to validate that the user has already solved a captcha within x minutes. The token is unique to a given user to reduce usefulness in brute force attacks. If a user doesn't exist, the token should always be invalid, which is what this is testing.